TypeScript.YAML-tmLanguage

```diff
@@ -466,12 +466,12 @@ repository:
 
   case-clause:
     name: case-clause.expr.ts
-    begin: '(?<!\.)\b(case|default)\b'
+    begin: '(?<!\.)\b(case|default(?=:))\b'
```

Comment:
This means case or (default followed by a lookahead for colon). It does not catch "case:".